contracts/HubPool.sol

```diff
@@ -478,7 +498,6 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
         uint256[] memory bundleLpFees
     ) internal {
         AdapterInterface adapter = crossChainContracts[chainId].adapter;
-        require(address(adapter) != address(0), "Adapter not set for target chain");
```

Comment:
do we need this check? elsewhere in `_executeRelayerRefundOnChain` for example, we don't check that the adapter address is non0